tests/integration/hive_metastore/test_migrate.py

```diff
@@ -221,7 +221,7 @@ def test_migrate_external_table_hiveserde_in_place(
         assert row["region"] == "us"
 
 
-@retried(on=[NotFound], timeout=timedelta(minutes=2))
+@retried(on=[NotFound, PermissionDenied], timeout=timedelta(minutes=2))
```

Comment:
What causes the `PermissionDenied` to occure? (Same as in https://github.com/databrickslabs/ucx/pull/2125#pullrequestreview-2168588716)

Reply:
it doesn't sound right. Permission denied is a real problem. What message are we retrying?

Reply:
Logs show this:
07:04 WARNING [databricks.labs.ucx.hive_metastore.table_migrate] Failed to migrate table hive_metastore.hiveserde_ctas_snfc.parquet_serde_snfc to ucx_c8ecu.hiveserde_ctas_snfc.parquet_serde_snfc: Failed to acquire a SAS token for list on /a/hiveserde_ctas_snfc/parquet_serde_snfc_ctas_migrated/_delta_log due to java.util.concurrent.ExecutionException: com.databricks.sql.managedcatalog.acl.UnauthorizedAccessException: PERMISSION_DENIED: Failed to authenticate with the configured service principal. Please contact your account admin to update the configuration. exceptionTraceId=e5ab7ff7-cee2-443e-9718-b4b46c6ed992
But the test passes when run from my laptop, whch suggests this is a transient issue.

Reply:
okay, it's related to token rotation. this PR is not neccessary